app/screens/snack_bar/index.tsx

```diff
@@ -87,9 +88,11 @@ const SnackBar = ({barType, componentId, onAction, sourceScreen}: SnackBarProps)
     const baseTimer = useRef<NodeJS.Timeout>();
     const mounted = useRef(false);
     const userHasUndo = useRef(false);
+    const {bottom} = useSafeAreaInsets();
 
     const config = SNACK_BAR_CONFIG[barType];
     const styles = getStyleSheet(theme);
+    const gestureRootChannelInfoAdjustment = useMemo(() => ({bottom: bottom || 8}), [bottom]);
```

Comment:
Hm, interesting -- that `marginBottom` should only be set if this is a tablet. 
It turns out, it's the height style for `gestureRoot` -- it was set to 80, and so the toast (which is only 56) sat at the top of that. @enahum I'm not sure how you want to deal with this... I think the "correct" way would be to remove the `height: 80` on `gestureRoot`, since that isn't needed. And now the toast sits where it should. However, this will affect the positions of all toasts. Their actual bottom will be where the gesture root's bottom is, rather than the gesture root's bottom + 80-56.
@matthewbirtch is this fine for you?
<table>
<tr>
<th>iPhone 13</th>
<th>iPhone SE</th>
</tr>
<tr>
<td>
<img width="226" alt="image" src="https://user-images.githubusercontent.com/1490756/201122418-24d8a355-2eac-4561-b47f-188f1db2e3b8.png">
</td>
<td><img width="189" alt="image" src="https://user-images.githubusercontent.com/1490756/201122370-385b2663-5abe-46a4-9003-fe26d58ebfdf.png"></td>
</tr>
</table>

Reply:
Screenshots look good here!